### PR TITLE
fix: alternate connector support for visual metrics

### PR DESCRIPTION
**Pull request by briangregoryholmes**

### CODE REVIEW

**Comment by medriscoll** [comment]
Just adding to this per a Slack thread with @ericpgreen2 -- will this allow the drop-down menu to be populated correctly in the Visual Metrics Editor?  (For projects that reference a CH table, the drop-down is currently null). 

image

**Comment by briangregoryholmes** [comment]
Yes

File: web-common/src/features/entity-management/resource-selectors.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export function prettyResourceKind(kind: string) {
 export function useResource<T = V1Resource>(
   instanceId: string,
   name: string,
-  kind: ResourceKind,
+  kind: ResourceKind | undefined,
```
Comment (ericpgreen2) [inline web-common/src/features/entity-management/resource-selectors.ts:47]: Can we avoid setting this to `undefined`? It feels like it breaks the semantics

**Review by ericpgreen2 — APPROVED** [review]
LGTM! Just one nice-to-have would be a memory of the last Model/Table selected, so that when you toggle back-and-forth between Model/Table, you go back to your previous state.

**Pull request merged**